MPM/mpm_settings.php

```diff
@@ -576,4 +576,33 @@ public function add_notice ($message, $type = 'notice')
 			}
 		}
 	}
+
+	/**
+	 * Restocks an order, a failed or cancelled order for example
+	 *
+	 * @param WC_Order $order
+	 */
+	public function restock_order(WC_Order $order)
```

Comment:
Deze kan ook weg, maar zou vervangen kunnen worden door `reduce_sales()`, zodat producten niet meer in rapportages zichtbaar is.